### PR TITLE
Adds Rails 7 support

### PR DESCRIPTION
**Pull request by pftg**

### CODE REVIEW

**Comment by donv** [comment]
The gemspec should reflect that actionpack >= 5.2 is required.

**Comment by donv** [comment]
...or did you add them back?

**Comment by pftg** [comment]
@donv this PR in progress. will cleanup a little for now. Maybe I'll split into 2 separate issues: drop unsupported ruby and rails 7 upgrade

File: capybara-screenshot-diff.gemspec [hunk 1]
```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "actionpack", ">= 4.2", "< 7"
+  spec.add_runtime_dependency "actionpack", ">= 4.2", "< 8"
```
Comment (pftg) [inline capybara-screenshot-diff.gemspec:23]: I just added max rails 8

**Comment by pftg** [comment]
@donv please check

**Comment by pftg** [comment]
@donv please re-check

**Comment by donv** [comment]
I merged the other PR.  Could you re-check and rebase this one?

**Comment by pftg** [comment]
@donv thanks, that PR had all required chagnes

**Pull request closed without merging**